fuel/app/config/config.php

```diff
@@ -196,18 +196,20 @@
 	/**
 	 * Cookie settings
 	 */
-	// 'cookie' => array(
+	'cookie' => array(
 		// Number of seconds before the cookie expires
 		// 'expiration'  => 0,
 		// Restrict the path that the cookie is available to
 		// 'path'        => '/',
 		// Restrict the domain that the cookie is available to
 		// 'domain'      => null,
 		// Only transmit cookies over secure connections
-		// 'secure'      => false,
+		'secure'      => $_SERVER['HTTPS'] ?? false,
 		// Only transmit cookies over HTTP, disabling Javascript access
 		// 'http_only'   => false,
-	// ),
+		// Samesite restrictions on cookie, options include Lax, Strict or None.
+		'same_site'   => (isset($_SERVER['HTTPS']) and $_SERVER['HTTPS']) ? 'None' : 'Strict',
```

Comment:
Same as above - although in the case of `same_site`, I feel like we could just stick to a default, discrete value of `None`.